platform/viewer/public/config/aws_static.js

```diff
@@ -0,0 +1,159 @@
+window.config = {
```

Comment:
This is a copy of the default.js configuration.  Generally, this monolithic configuration isn't a great idea, particularly mixing in data sources etc.  Need to figure out some better options for this longer term.